packages/storybook8/package.json

```diff
@@ -48,22 +48,22 @@
     "@babel/preset-env": "7.23.9",
     "@mdx-js/react": "^1.6.22",
     "@storybook/addon-webpack5-compiler-babel": "^3.0.3",
-    "@storybook/core-events": "^8.3.5",
```

Comment:
Is there a storybook github issue with this problem? I can't find one on their github

Reply:
OK has been logged once or twice lol
![image](https://github.com/user-attachments/assets/cd4d6cb0-4241-47b6-83ec-1276f5c3cd5b)